src/flight-search/Client.js

```diff
@@ -135,15 +142,23 @@ FlightSearchClient.prototype = {
             departureAirportCode: leg.departureAirportCode,
             arrivalCityCode: leg.arrivalCityCode,
             arrivalAirportCode: leg.arrivalAirportCode,
-            outboundDate: leg.outboundDate,
+            outboundDate: leg.outboundDate
           };
-        }),
+        })
       },
       offset: this.processedFaresCount,
       paymentMethodIds: this.paymentMethodIds,
-      providerTypes: this.providerTypes,
-    }
+      providerTypes: this.providerTypes
+    };
   },
+
+  fetchTripsParams: function() {
+    return {
+      currencyCode: this.currency.code,
```

Comment:
Can we assign var for `this`

Reply:
like `var self = this` ?
why is it needed?

Reply:
Not actually needed, it's optional. Just making it DRY and prettier.

Reply:
i see, is the convention `var self = this` ?